src/content/protocols/pwn/ethereum.md

```diff
@@ -0,0 +1,197 @@
+| PWNHub | setTag | Sets tag for a specified contract. Having a tag makes a contract valid in the protocol. Malicious actor could make his own contract valid and act as for example the Loan contract. Users would have to manually specify the usage of this contract. | TimelockController (Protocol) |
+| PWNHub | setTags | Same as setTag. This function calls setTag multiple times. | TimelockController (Protocol) |
+| PWNLOAN | mint | Mints a LOAN token. LOAN token represents a loan in the protocol for the lender. It is required to claim repayment or collateral. Malicious actor could mint worthless LOAN tokens and try to use them as a collateral for loans. | Loan contract specified in an offer |
+| PWNLOAN | burn | Burns a LOAN token.LOAN token represents a loan in the protocol for the lender. It is required to claim repayment or collateral. Malicious actor could burn other LOAN tokens and make it impossible to claim loans. | Loan contract specified in an offer |
+| PWNRevokedNonce | revokeNonce | Revokes nonce for an offer. Used to revoke an offchain signed offer on behalf of an owner. Malicious actor could revoke offers that the user wants to be valid. | Contract with a valid tag in PWNHub |
+| PWNRevokedNonce | revokeNonce | Revokes nonce for an offer. Used to revoke an offchain signed offer on behalf of an owner. Malicious actor could revoke offers that the user wants to be valid. | Contract with a valid tag in PWNHub |
+| PWNSimpleLoan | createLOAN | Creates a loan in the PWN Protocol. The check in this function is for a valid lenderSpec check in case the caller is a lender. There's also a reentrancy protection modifier. | Lender |
+| PWNSimpleLoan | repayLOAN | Repays a running loan. There's a reentrancy protection modifier to make sure you can't repay more than once. | Borrower |
+| PWNSimpleLoan | claimLOAN | Claims repayment or defaulted collateral. There's a reentrancy protection modifier to make sure you can't claim more than once. | Lender |
+| PWNSimpleLoan | tryClaimRepaidLOAN | Tries to claim loan for a lender. The check is this function is to make sure this function can only be called by the Loan contract. Malicious actor could initiate the claim without the Loan contract. Note that it's not possible to claim through this function if the original loan lender has changed and therefor malicious actor cannot claim the assets to its wallets. | Loan contract |
+| PWNSimpleLoan | makeExtensionProposal | Makes an onchain proposal for loan extension. The borrower can create a proposal for extension. Malicious actor could create an extension proposal for any loan. | Proposer |
+| PWNSimpleLoan | extendLOAN | Extends a loan based on extension proposal. Both the lender and the borrower can call this function to extend a loan they're part of if the other party has signed the extension proposal. The checks in this function are to distinguish two possible states (lender or borrower calling the function). | Lender or borrower |
```

Comment:
we dont include user actions in protocols as part of the permissions, we didnt include user actions in aave, spark and pendle e.g
```suggestion
```